### PR TITLE
[issue 100] - Generate the RHSSO (7.6.z) Operator provisioner Java model

### LINKED ISSUE #100: Add RHSSO 7.6.z Operator based provisioning tooling
**Is your feature request related to a problem? Please describe.**
An operator based provisioner for RHSSO 7.4.5 is needed to run interoperability tests that involve RHSSO 7.4.5.  

The current [Keycloak (Realm Import) operator based provisioner](https://github.com/Intersmash/intersmash/blob/main/tools/intersmash-tools-provisioners/src/main/java/org/jboss/intersmash/tools/provision/openshift/KeycloakRealmImportOperatorProvisioner.java) - which is the latest on the community side - is not compatible with [the old one](https://github.com/Intersmash/intersmash/blob/main/tools/intersmash-tools-provisioners/src/main/java/org/jboss/intersmash/tools/provision/openshift/KeycloakOperatorProvisioner.java) and [its model](https://github.com/Intersmash/intersmash/tree/main/tools/intersmash-tools-provisioners/src/main/java/org/jboss/intersmash/tools/provision/openshift/operator/keycloak) - which is based on old CRDs- despite those still being the latest on the product side.  

**Describe the solution you'd like**
- [ ] Include CRDs from the [old (archived) Keycloak operator](https://github.com/keycloak/keycloak-operator)
- [ ] replace current model with generated JOSDK
- [ ] adjust the currently iplemented provisioning logic
- [ ] adjust the provisioner test to verify it can be used in the CI e2e tests, productized variant 

**Describe alternatives you've considered**
Rework internal tests that are currently using RHSSO Operator based provisioner in order to use a different one but that would require for more work and time.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by fabiobrz**
## Description
Fix #100 

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [x] Pull Request contains a description of the changes
 - [x] Pull Request does not include fixes for multiple issues/topics
 - [x] Code is self-descriptive and/or documented
 - [x] I have implemented unit tests to cover my changes
 - [x] I tested my code in OpenShift

### CODE REVIEW

File: testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/operator/OperatorSubscriptionTestCase.java [hunk 1]
```diff
@@ -20,18 +20,21 @@
...
+import org.jboss.intersmash.tools.application.openshift.KeycloakRealmImportOperatorApplication;
+import org.jboss.intersmash.tools.application.openshift.RhSsoOperatorApplication;
 import org.jboss.intersmash.tools.application.openshift.WildflyOperatorApplication;
 import org.jboss.intersmash.tools.junit5.IntersmashExtension;
 import org.jboss.intersmash.tools.provision.openshift.ActiveMQOperatorProvisioner;
 import org.jboss.intersmash.tools.provision.openshift.HyperfoilOperatorProvisioner;
 import org.jboss.intersmash.tools.provision.openshift.InfinispanOperatorProvisioner;
 import org.jboss.intersmash.tools.provision.openshift.KafkaOperatorProvisioner;
-import org.jboss.intersmash.tools.provision.openshift.KeycloakOperatorProvisioner;
+import org.jboss.intersmash.tools.provision.openshift.KeycloakRealmImportOperatorProvisioner;
```
Comment (tommaso-borgato) [inline testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/operator/OperatorSubscriptionTestCase.java:36]: @fabiobrz shouldn't KeycloakRealmImportOperatorProvisioner be renamed as well? e.g. to KeycloakOperatorProvisioner?

Comment (fabiobrz) [reply testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/operator/OperatorSubscriptionTestCase.java:36]: You seem to be right, thanks for catching. Fixing!

Comment (fabiobrz) [reply testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/operator/OperatorSubscriptionTestCase.java]: Done.

**Pull request merged**